src/pages/gas.tsx

```diff
@@ -0,0 +1,533 @@
+)
+
+export const StyledCard = (props: ComponentPropsWithRef<typeof Card>) => (
+  <Card
+    flex="1 1 30%"
+    minW="280px"
+    maxW={{ base: "full", lg: "46%" }}
+    p={6}
+    {...props}
+  />
+)
+
+const H2 = (props: HeadingProps) => (
+  <Heading fontSize={{ base: "2xl", md: "2rem" }} lineHeight={1.4} {...props} />
+)
+
+const H3 = (props: HeadingProps) => (
+  <Heading
+    as="h3"
+    fontSize={{ base: "xl", m: "2xl" }}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const GasPage = ({ data }: PageProps<Queries.GasPageQuery>) => {
+  const { t } = useTranslation()
+
+  const benefits = [
+    {
+      emoji: "🪪",
+      description:
+        "Gas keeps Ethereum sybil-resistant by preventing malicious actors from overwhelming the network with fraudulent activities.",
+    },
+    {
+      emoji: ":money_with_wings:",
+      description:
+        "Because computation costs gas, spamming Ethereum with expensive transactions, either accidentally and maliciously, is financially disencentivised.",
+    },
+    {
+      emoji: ":hourglass_flowing_sand:",
+      description:
+        "A hard-limit on the amount of computation that can be done at any one time prevents Ethereum from being overwhelmed, helping to ensure the network is always accessible.",
+    },
+  ]
+
+  const heroContent = {
+    title: "Network fees",
+    header: "Network fees",
+    image: getImage(data.infrastructure)!,
+    alt: "",
+    buttons: [
+      {
+        content: "What is gas?",
+        toId: "what-is-gas",
+        matomo: {
+          eventCategory: "gas hero buttons",
+          eventAction: "click",
+          eventName: "wgat is gas",
+        },
+      },
+    ],
+  }
+
+  return (
+    <Page>
+      <PageMetadata
+        title="Gas fees on Ethereum: how do they work?"
+        description="Learn about gas on Ethereum: how they work and how to pay less in gas fees"
+      />
+      <Box background="layer2Gradient" width="full">
+        <Box pb={8}>
+          <PageHero
+            content={{
+              subtitle: (
+                <Box>
+                  <Text mb={0}>Network fees on Ethereum are called gas.</Text>
+                  <Text>Gas is the fuel that powers Ethereum.</Text>
+                </Box>
+              ),
+              ...heroContent,
+            }}
+          />
+        </Box>
+      </Box>
+      <>
+        <Content mb={{ base: 16, md: 16, lg: 32 }} mt={16}>
+          <Flex
+            direction={{ base: "column", lg: "row" }}
+            align={{ base: "center", lg: "flex-start" }}
+            w="full"
+          >
+            <Box flex="60%" w="full" mr={{ base: "auto", lg: 2 }}>
+              <InfoBanner mb={8} title="Summary">
+                <UnorderedList>
+                  <ListItem>
+                    Every transaction on Ethereum requires a small form of
+                    payment to process
+                  </ListItem>
+                  <ListItem>These fees are known as ‘gas’ fee.</ListItem>
```

Comment:
```suggestion
                  <ListItem>These fees are known as ‘gas’ fee</ListItem>
```

Reply:
other items dont have a dot at the end of the sentence, so either add everywhere or remove here

Reply:
Nice catch!